certbot-nginx/local-oldest-requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Remember to update setup.py to match the package versions below.
-acme[dev]==1.0.0
+-e acme[dev]
```

Comment:
We should also update the nginx plugin's `setup.py` file since we're not going to be testing against 1.0 anymore.

Reply:
Done.